packages/runtime/container-runtime/src/gc/gcSummaryStateTracker.ts

```diff
@@ -50,6 +50,10 @@ export class GCSummaryStateTracker {
 	// Tracks whether there was GC was run in latest summary being tracked.
 	private wasGCRunInLatestSummary: boolean;
 
+	// Tracks the count of data stores whose state updated since the last summary, i.e., they went from referenced
+	// to unreferenced or vice-versa.
+	public stateUpdatedDSCountSinceLastSummary: number = 0;
```

Comment:
Can we get rid of the word `state` in the front of this and all other related names?